server/sonar-server/src/main/java/org/sonar/server/source/index/SourceLineIndexer.java

```diff
@@ -48,15 +50,28 @@ public SourceLineIndexer(DbClient dbClient, EsClient esClient) {
     this.dbClient = dbClient;
   }
 
+  public void index(final String projectUuid){
+    super.index(new IndexTask() {
+      @Override
+      public long index(long lastUpdatedAt) {
```

Comment:
the index method name is misleading with the indexer index method. I think it should be different.